backend/app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class UserMailer < ApplicationMailer
```

Comment:
I would probably create `AccountMailer`

Reply:
I don't know, what do you think?

Reply:
On the second thought, actually `UserMailer` is better, I'm not sure if only account owner should get that email or all users of the account? cc @agnessa

Reply:
Yeah, I was thinking the same. `UserMailer` makes more sense for both cases, sending to single account owner and sending to all users.